.changes/unreleased/charts-operator-Deprecated-20250226-171814.yaml

```diff
@@ -0,0 +1,10 @@
+project: charts/operator
+kind: Deprecated
+body: |
+  The gcr.io/kubebuilder/kube-rbac-proxy container is being deprecated. Redpanda
+    operator helm chart would stop deploying side car. The ports are preserved,
+    so the prometheus rules don't need to be adjusted.
+
+    Reference
+    https://github.com/kubernetes-sigs/kubebuilder/discussions/3907
```

Comment:
```suggestion
    `gcr.io/kubebuilder/kube-rbac-proxy container` is deprecated and has been removed from the Redpanda
    operator helm chart. The same ports will continue to serve metrics using kubebuilder's built in RBAC.
    Any existing prometheus rules don't need to be adjusted.

    For more details see: https://github.com/kubernetes-sigs/kubebuilder/discussions/3907
```

I think this should also be a "Removed" instead of deprecated because we removed it entirely.

Reply:
Ok I will change to Removed